app/controllers/alerts_list_controller.rb

```diff
@@ -16,6 +17,15 @@ def index
     redirect_to :action => 'show'
   end
 
+  def class_icons
+    res = {}
+    icons.each do |k, _val|
+      file_name = "100/#{k.to_s.underscore}.png"
```

Comment:
uploaded a new version, still haven't convinced it to work tough...
I keep getting:
```
[----] F, [2017-02-28T20:19:32.671115 #5014:2f04e3c] FATAL -- : Error caught: [NoMethodError] undefined method `listicon_image' for #<Draper::CollectionDecorator:0x0000000b7d4280>
/home/mtayer/dev/manageiq-ui-classic/app/controllers/alerts_list_controller.rb:22:in `block in class_icons'
```
I'll try to rebase the client code as well tomorrow